FirebaseFunctions.podspec

```diff
@@ -28,6 +28,9 @@ iOS SDK for Cloud Functions for Firebase.
   s.source_files = 'Functions/FirebaseFunctions/**/*'
   s.public_header_files = 'Functions/FirebaseFunctions/Public/*.h'
 
+  s.frameworks = [
+    'UIKit',
```

Comment:
I wasn't sure if this was strictly needed; UIKit is only in example code which I wasn't sure was included in the pod.